simplivity/resources/policies.py

```diff
@@ -199,3 +199,21 @@ def get_rule(self, rule_id):
         """
         resource_uri = "{}/{}/rules/{}".format(URL, self.data["id"], rule_id)
         return self._client.do_get(resource_uri)
+
+    def rename(self, name, timeout=-1):
```

Comment:
Can we change the variable name to new_name?

Reply:
Updated the variable name to "new_name"